packages/core/src/controllers/subscriber.ts

```diff
@@ -236,6 +236,7 @@ export class Subscriber extends ISubscriber {
     };
     this.logger.debug(`Outgoing Relay Payload`);
     this.logger.trace({ type: "payload", direction: "outgoing", request });
+    const shouldThrow = opts?.internal?.throwOnFailedPublish;
```

Comment:
this is true by default now, right? Would it cause issues in wallets that upgrade to this version? 🤔